gruntfile.js

```diff
@@ -166,6 +166,8 @@ function shell_run_container() {
       -p 8000:8000 \
       -p 9000:9000 \
       -p 9443:9443 \
+      --net caddy_proxy \
```

Comment:
shouldn't this be added if the feature flag is enabled?

Reply:
oh this shouldn't be here at all - its how I do everything, but it totally a very future-proposal :D